### PR TITLE
Sets

### PR DESCRIPTION
**Pull request by lpil**

### CODE REVIEW

File: concepts/sets/about.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+// -> [1, 2, 3]
+```
+
+You can create and populate sets with `set.new`, `set.insert` and `set.delete`.
+
+```gleam
+set.new()
+// -> set.from_list([])
+
+let eighty_eight = set.from_list(88)
```
Comment (giacomocavalieri) [inline concepts/sets/about.md:23]:
```suggestion
let eighty_eight = set.from_list([88])
```

File: concepts/sets/introduction.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+// -> [1, 2, 3]
+```
+
+You can create and populate sets with `set.new`, `set.insert` and `set.delete`.
+
+```gleam
+set.new()
+// -> set.from_list([])
+
+let eighty_eight = set.from_list(88)
```
Comment (giacomocavalieri) [inline concepts/sets/introduction.md:23]:
```suggestion
let eighty_eight = set.from_list([88])
```

File: exercises/concept/gotta-snatch-em-all/.docs/instructions.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+// -> ["Scientuna"]
+```
+
+## 5. All of the cards
+
+Do you and your friends collectively own all of the Blorkemon™️ cards?
+
+Implement `totalCards`, which takes a list of collections and returns the total number of different cards in the all of the collections.
+
+```elm
```
Comment (giacomocavalieri) [inline exercises/concept/gotta-snatch-em-all/.docs/instructions.md:61]:
```suggestion
```gleam
```

File: exercises/concept/gotta-snatch-em-all/.docs/instructions.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+```gleam
+boring_cards([set.from_list(["Scientuna"]), set.from_list(["Newthree", "Scientuna"])])
+// -> ["Scientuna"]
+```
+
+## 5. All of the cards
+
+Do you and your friends collectively own all of the Blorkemon™️ cards?
+
+Implement `totalCards`, which takes a list of collections and returns the total number of different cards in the all of the collections.
```
Comment (giacomocavalieri) [inline exercises/concept/gotta-snatch-em-all/.docs/instructions.md:59]:
```suggestion
Implement `totalCards`, which takes a list of collections and returns the total number of different cards in all of the collections.
```

File: exercises/concept/gotta-snatch-em-all/.docs/instructions.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+```elm
+totalCards [set.from_list(["Scientuna"]), set.from_list(["Newthree", "Scientuna"])]
+// -> 2
+```
+
+## 6. Shiny for the win
+
+You nephew is coming to visit you soon, and you feel like impressing him.
```
Comment (giacomocavalieri) [inline exercises/concept/gotta-snatch-em-all/.docs/instructions.md:68]:
```suggestion
Your nephew is coming to visit you soon, and you feel like impressing him.
```

File: exercises/concept/gotta-snatch-em-all/.docs/introduction.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+// -> [1, 2, 3]
+```
+
+You can create and populate sets with `set.new`, `set.insert` and `set.delete`.
+
+```gleam
+set.new()
+// -> set.from_list([])
+
+let eighty_eight = set.from_list(88)
```
Comment (giacomocavalieri) [inline exercises/concept/gotta-snatch-em-all/.docs/introduction.md:25]:
```suggestion
let eighty_eight = set.from_list([88])
```

**Comment by giacomocavalieri** [comment]
Love the Blorkemon™️ exercise, feels right at home 😆
Sorry, if I've have made the same comment for many files that all look the same, I guess some are generated automatically but I couldn't tell which ones

**Comment by lpil** [comment]
Right? I love the copy for this one 💜 

Thank you so much!!

**Pull request merged**